web/models/product_graph.py

```diff
@@ -78,6 +81,23 @@ def build_stopword_index(self):
             add_to_search_index(index, doc_id, stopword)
         return index
 
+    def build_nutrition_index(self, nutrition):
+        index = build_search_index()
+        for nutrition in nutrition or []:
```

Comment:
This variable name re-use isn't ideal; there might be a better naming scheme to use for the 'list of nutrional data items' (function argument) vs the 'nutrition data to be added to the index' (loop variable).

Reply:
(done)